pages/docs/tutorials/native/using-command-line-compiler.md

```diff
@@ -0,0 +1,54 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Hello Kotlin/Native using Command Line Compiler"
+description: "A look at how to compileKotlin/Native applications using the command line compiler"
+authors: 
+  - Hadi Hariri
+date: 2020-01-15
+---
+
+<!--- To become a How-To. Need to change type to new "HowTo" --->
+
+
+## Obtaining the Compiler
```

Comment:
Couldn't find anything about obtaining the compiler in this section. Mabye add a couple of sentences explaining what is it (like: a CLU util, we ship it in Kotlin distrib with each release, etc.) and where to get it.

Reply:
Added.